packages/documentation/src/content/docs/resources/get-involved.mdx

```diff
@@ -4,27 +4,91 @@ title: Get involved
 
 import { LinkOut } from '@interledger/docs-design-system'
 
-## Participation
+Welcome to the Rafiki community! Whether you're a seasoned developer or just getting started with open source, there are many ways to contribute to the Rafiki project. This guide will help you find your place in our vibrant ecosystem.
 
-- Begin a <LinkOut href="https://github.com/interledger/rafiki/discussions">new discussion or contribute to existing discussions</LinkOut> in the Rafiki GitHub repo.
-- Join the <LinkOut href="https://interledger.slack.com">`interledger.slack.com`</LinkOut> workspace and ask any questions you have in the `#rafiki` channel.
-- Try [running the system locally](/integration/playground/overview) and ask any questions you have in the #rafiki Slack channel.
+## 🚀 Getting started
 
-For more details about spec contributions and development, visit <LinkOut href="https://github.com/interledger/rafiki/blob/main/.github/contributing.md">`contributing.md`</LinkOut>.
+### New to Rafiki?
 
-## Coding
+- **Explore the Project**: Browse the <LinkOut href="https://github.com/interledger/rafiki">Rafiki repository</LinkOut> to understand the codebase structure and recent activity.
+
+- **Join Our Community**:
+
+  - Connect with us on <LinkOut href="https://interledger.slack.com">`interledger.slack.com`</LinkOut> in the `#rafiki` channel for real-time discussions
```

Comment:
So I used this link in the Open Payments Get Involved page: 
https://communityinviter.com/apps/interledger/interledger-working-groups-slack

No one has told me not to, and it keeps people from seeing that weird "sign in with your Coil account" message.